### ISSUE TITLE
[cli] Cannot add groups as project roles using CLI

### ISSUE THREAD

**Issue by dbason** [rancher/rancher#31430]
**What kind of request is this (question/bug/enhancement/feature request):** Bug

**Steps to reproduce (least amount of steps as possible):**
Enable openLdap Auth
Try to add an ldap group as a member to a project `rancher project add-member-role group1 project-member`

**Result:**
```
FATA[0001] Bad response statusCode [422]. Status [422 Unprocessable Entity]. Body: [baseType=error, code=InvalidBodyContent, message=must target a user [userId]/[userPrincipalId] OR a group [groupId]/[groupPrincipalId]] from [https://rancherurl/v3/projectroletemplatebindings]
```

**Expected Result:**
ProjectRoleTemplateBinding is created with grou
**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): Replicated on rancher 2.4.8

gz#14974
gz#15419

**Comment by deniseschannon** [comment]
@dramich I've assigned you to help answer any questions that QA might have

**Comment by dramich** [comment]
This is available in CLI v2.4.11-rc3

**Comment by aaronRancher** [comment]
The bug fix is verified with Rancher server `rancher\rancher:master-head` `version eda2a07` and CLI `v2.4.11-rc3`

Steps:
Enable openLdap Auth
Create group
From CLI, add an ldap group as a member to a project `./rancher project add-member-role testuser2 project-member`

Result:
Project member is added to project
![Screen Shot 2021-03-08 at 4 55 55 PM](https://user-images.githubusercontent.com/57409052/110398305-d5484180-8030-11eb-84ed-84f6c0b168fe.png)
ProjectRoleTemplateBinding is created for user:
```
"name": "prtb-fnk2b",
"namespaceId": null,
"projectId": "c-r26zr:p-6bk72",
"roleTemplateId": "project-member",
"type": "projectRoleTemplateBinding",
"userId": "u-6fukyez32k",
"userPrincipalId": "local://u-6fukyez32k",
"uuid": "9c1a4b29-6b71-449d-bdc0-9136b3583cb1"
```
